client/package.json

```diff
@@ -24,6 +25,7 @@
         "jwt-decode": "^2.2.0",
         "react": "^18.2.0",
         "react-colorful": "^5.6.1",
+        "react-country-region-selector": "^3.6.1",
```

Comment:
Do we still need this react country region selector npm?

Reply:
i removed it